cocos/ui/editbox/edit-box.ts

```diff
@@ -492,6 +492,9 @@ export class EditBox extends Component {
     public _editBoxEditingDidBegan (): void {
         ComponentEventHandler.emitEvents(this.editingDidBegan, this);
         this.node.emit(EventType.EDITING_DID_BEGAN, this);
+        if (this._impl) {
+            this._impl._dirtyFlag = true;
```

Comment:
Should we consider marking the dirty flag only when the text changes?

Reply:
impl doesn't need to be updated before start editing.